### PR TITLE
Commit copied egeria-connector files from fybrik repo.

### PR DESCRIPTION
**Pull request by revit13**
This PR exctracts egeria-connector related files from fybrik repo to to the egeria-connector repo. It contains the required changes to the chart and the source to deploy them outside the fybrik repo.

Signed-off-by: Revital Sur <eres@il.ibm.com>

### CODE REVIEW

**Review by ronenkat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
-# egeria-connector
+# Egeria Connector
+
+test1
```
Comment (rohithdv) [inline README.md]: can be removed?

Comment (revit13) [reply README.md]: sure tnx

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,45 @@
+ROOT_DIR := .
+DOCKER_NAME = egr-connector
+
+include $(ROOT_DIR)/hack/make-rules/docker.mk
+
+KUBE_NAMESPACE ?= fybrik-system
+
+.PHONY: docker-build
+docker-build: build
+	make docker-build -C connector-src
```
Comment (rohithdv) [inline Makefile:10]: this is a recursive statement?

Comment (rohithdv) [inline Makefile:10]: build is getting called twice ?

**Review by rohithdv — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (revit13) [reply Makefile:10]: fixed thanks

**Pull request merged**